cli/src/build/onboarding/build-log.ts

```
@@ -0,0 +1,59 @@
+// line instead of overwriting another), drop ANSI/escape/other C0 control bytes
+// (the viewer re-applies its own ✔/✖/⚠ colouring), and keep tabs + interior
+// blank lines. The result is plain text whose display width matches its content,
+// so the VT never overwrites and Ink always clears correctly.
+//
+// NOTE: this lives on the ONBOARDING viewer path only — the plain `capgo build`
+// console path keeps fastlane's colours (a real terminal renders them fine), so
+// request.ts is deliberately left untouched.
+
+// CSI sequences: ESC [ ... final-byte — SGR colours, cursor moves, erase-line, etc.
+// eslint-disable-next-line no-control-regex
+const CSI_RE = /\x1B\[[0-9;?]*[ -/]*[@-~]/g
```

ESLint

[error] Unexpected obscure character range. The characters of ' -/' (U+0020 - U+002f) are not obvious.

(regexp/no-obscure-range)